### DISCUSSION TITLE
Would like to add additional required fields on the Bullhorn OSCP job application

### DISCUSSION THREAD

**Discussion by meganjoell** [bullhorn/career-portal#440] [General]
This is far out of my wheelhouse, my boss would like to add the following fields required to apply to jobs and I am looking for some help! 

The additional required fields we would like to add are below:
Address, city, state, zip
Phone number
Are you eligible to work in the US
the website is https://clutchnow.com/jobs 

I have reached out to bullhorn and was directed to the forum and cannot find a straight answer. I am willing to pay a developer at this point because I am so lost.

**Comment by charlesabarnes** [comment]
Hmm, I don't believe you should've been directed here.  This platform is for getting assistance with code-related issues for the portal.   I suggest checking out Bullhorn's marketplace partners for some of their custom solutions.  https://www.bullhorn.com/marketplace/?category=source-talent

**Comment by pbearne** [comment]
Hi @Meganjoell

Paul from Matador Jobs. We are one of the partners you will find on the link Charles provided
We can do this out of the box with our plugin :-)
reach for more info
paul@matadorjobs.com
